vpr/src/draw/draw.cpp

```diff
@@ -440,15 +440,16 @@ void update_screen(ScreenUpdatePriority priority, const char* msg, enum pic_type
 }
 
 void toggle_window_mode(GtkWidget* widget, ezgl::application* app) {
+    (void)widget;
```

Comment:
The convention we usually use in VPR for unused parameters is to comment out the parameter name. So instead of casting the parameter to void, we would do something like:
```cpp
 void toggle_window_mode(GtkWidget* /*widget*/, ezgl::application* /*app*/) { 
```
which also avoids the compiler warning.